src/main/java/org/skriptlang/skript/bukkit/itemcomponents/blocking/elements/ExprReductionBase.java

```diff
@@ -0,0 +1,90 @@
+package org.skriptlang.skript.bukkit.itemcomponents.blocking.elements;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.blocking.BlockingExperimentalSyntax;
+import org.skriptlang.skript.bukkit.itemcomponents.blocking.DamageReductionWrapper;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+@Name("Damage Reduction - Base Amount")
+@Description("""
+	The base amount of damage to block when the item blocks an attack.
```

Comment:
I think it's better to say "flat amount" since base implies it's going to be modified